Client/Frontend/Browser/Tab.swift

```diff
@@ -737,6 +766,13 @@ class TabWebView: BraveWebView, MenuHelperInterface {
 
         return super.value(forUndefinedKey: key)
     }
+
+    private func getCurrentSelectedText(callback: @escaping (String?) -> Void) {
```

Comment:
Maybe convert to async/await? We officially use Xcode 13.2 and higher now

Reply:
Changing this seems somewhat extensive. I only worked a little bit with async/await and there are things happening in `evaluateSafeJavaScript` that are beyond my understanding for the moment. Though I agree it would be nice to switch to async/await for these methods I would have to spent a deal of time to make it work.

Reply:
no worries, your code is completely viable, no async await code is introduced to our codebase yet, we can take time